### PR TITLE
Noble 24.1.1 release

### PR DESCRIPTION
**Pull request by holmanb**
## Merge type

- [ ] Squash merge using "Proposed Commit Message"
- [x] Rebase and merge unique commits. Requires commit messages per-commit each referencing the pull request number (#<PR_NUM>)

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,13 @@
+cloud-init (24.1.1-0ubuntu0) UNRELEASED; urgency=medium
```
Comment (TheRealFalcon) [inline debian/changelog]: This is one of the few places that `new_upstream_snapshot.py` can guess wrong. Version number should be `24.1.1-0ubuntu1`.

**Review by TheRealFalcon — CHANGES_REQUESTED** [review]
Just need a version number change here.

File: debian/changelog (hunk 1 above)
Comment (holmanb) [reply debian/changelog]: > This is one of the few places that new_upstream_snapshot.py can guess wrong

Is that something that can be fixed?

**Comment by holmanb** [comment]
> @holmanb I'm still seeing the old version number. In particular the `0ubuntu0` should be `0ubuntu1`

oops, fixed now
```shell
$ head debian/changelog
cloud-init (24.1.1-0ubuntu1) noble; urgency=medium

  * Upstream snapshot based on 24.1.1.
    List of changes from upstream can be found at
    https://raw.githubusercontent.com/canonical/cloud-init/24.1.1/ChangeLog
    - Bugs fixed in this snapshot: (LP: #2056439, #2056460, #2055077)
      (LP: #2056194)

 -- Brett Holman <brett.holman@canonical.com>  Mon, 11 Mar 2024 21:09:37 -0600
```

File: debian/changelog (hunk 1 above)
Comment (TheRealFalcon) [reply debian/changelog]: Probably. I think in the past we always used `ubuntu/devel` even through feature freeze, so the usage is a little different.

**Review by TheRealFalcon — APPROVED** [review]
LGTM!

**Pull request merged**